### DISCUSSION TITLE
Wait for the network to be configured error

### DISCUSSION THREAD

**Discussion by tai5555** [cockpit-project/cockpit#18370] [Q&A]
Hi! 
      Can you please guide me how I can address this error. Due to this the virtual machines cannot ping the gateway! Please guide me. Thanks
![cockpit_ERROR](https://user-images.githubusercontent.com/97503378/220019044-fe4f641a-9341-4262-85e4-18174771d062.PNG)

**Comment by jelly** [comment]
This is not a cockpit issue, this is a service of systemd, this [issue](https://github.com/systemd/systemd/issues/2154) seems to have more information.
